doc/api/esm.md

```diff
@@ -749,7 +749,7 @@ changes:
 * `context` {Object}
   * `conditions` {string\[]} Export conditions of the relevant `package.json`
   * `importAssertions` {Object}
-  * `parentURL` {string|undefined} The module importing this one, or undefined
+  * `parentUrl` {string|undefined} The module importing this one, or undefined
```

Comment:
oof, why is URL - an initialism - not capitalized? Url is a Java-ism :-/

Reply:
just saw the discussion above: https://github.com/nodejs/node/pull/43164#discussion_r887326916